src/Shared/Server.cs

```diff
@@ -223,6 +228,7 @@ public enum DataToLoad
 		Exp = 0x80,
 		Dialogues = 0x100,
 		Shops = 0x200,
+		Help = 0x400,
```

Comment:
Another plus for changing how we load stuff, no conflicts for two people adding new dbs at the same time anymore^^